src/pages/FAQ.tsx

```diff
@@ -0,0 +1,182 @@
+import { useQuery } from 'react-query';
+import { useState, useEffect } from 'react';
+import { Text, VStack, HStack, Button} from 'native-base';
+import { getFAQs, setFAQs } from '../xplat/queries/faq';
+import { FAQCollection, FAQItem } from '../xplat/types/faq';
+import FAQCardBox, { EditFAQCardBox } from '../components/FAQCardBox';
+
+const FAQ = () => {
+  const {data, isError, isLoading} = useQuery( 'faq_collection', () => getFAQs());
+  if (isError)
+  {
+    return (
+      <VStack alignItems='center' space={2}>
+        <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+        <Text>Error loading data.</Text>
+      </VStack>
+    );
+  }
+
+  if (isLoading || data === undefined)
+  {
+    return (
+      <VStack alignItems='center' space={2}>
+        <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      
+        <Text>Loading...</Text>
+
+      </VStack>
+    );
+  }
+
+  return (
+    <VStack alignItems='center' space={2}>
+      <img
+        src={process.env.PUBLIC_URL + '/logo.jpg'}
+        style={{
+          maxWidth: '200px',
+          height: 'auto'
+        }}
+        alt='logo'
+      />
+      <Text variant='header' bold fontSize='3xl'>Frequently Asked Questions</Text>
+      
+      {
+        data.map( (value, index) => 
+          <FAQCardBox key={index} question={value.question} answer={value.answer}/>)
+      }
+
+    </VStack>
+  );
+};
+
+export const EditFAQ = () => {
+  const [FAQCards, setFAQCards] = useState<FAQCollection>();
+  const [update, setUpdate] = useState<number>(-1);
+  const [addingNewCard, setAddingNewCard] = useState<boolean>(false);
+  const [changesExist, setChangesExist] = useState<boolean>(false);
+  const {data, isError, isLoading, refetch} = useQuery( 'faq_collection', () => getFAQs());
+
+  const addCard = (question: string, answer: string) => {
+    if (FAQCards === undefined)
+      return;
+    
+    const tempList = FAQCards;
+    tempList.push( {
+      question: question,
+      answer: answer
+    });
+    setAddingNewCard(false);
+    setChangesExist(true);
+    setFAQCards(tempList);
+  };
+
+  const removeCard = (index: number) => {
+    if (FAQCards === undefined)
+      return;
+    if (index >= FAQCards.length)
+      return;
+    const tempList = FAQCards;
```

Comment:
is this a reference copy? if so, we prolly need to do the splice on the new list we create, and then set it